resource/airflow_dag_pipeline.yaml

```diff
@@ -98,59 +113,35 @@ jobs:
           - name: image
 
   - aggregate:
-    - task: update-to-test-iam-role-name
+    - task: -list-policy
```

Comment:
starting with `-` looks weird even if it's valid

Reply:
yep, sorted